templates/boarding-pass/templates/main.tpl

```diff
@@ -0,0 +1,112 @@
+
+    {{template "table-cell-paragraph" (extendDict $props "BorderBottomSize" 1 "Margin" "0" "Align" "left" "Font" "helvetica-bold" "FontSize" 12 "Text" (printf "%s / %s" .From.City .From.Code)) }}
+    {{template "table-cell-paragraph" (extendDict $props "Text" (printf "%s / %s" .Destination.City .Destination.Code)) }}
+
+    <table-cell rowspan="5" background-color="#ffffff">
+        <image src="path('templates/res/baggage.png')" fit-mode="fill-width" margin="20"></image>
+    </table-cell>
+
+    {{template "table-cell-paragraph" (extendDict $props "Colspan" 1 "BorderBottomSize" 0 "Margin" "5 0 0 0" "Font" "helvetica" "FontSize" 10 "Text" "Flight") }}
+    {{template "table-cell-paragraph" (extendDict $props "Text" "Gate") }}
+    {{template "table-cell-paragraph" (extendDict $props "Text" "Class") }}
+    {{template "table-cell-paragraph" (extendDict $props "Text" "Seat") }}
+
+    {{template "table-cell-paragraph" (extendDict $props "BorderBottomSize" 1 "Margin" "0" "Font" "helvetica-bold" "FontSize" 12 "TextColor" "#ff0000" "Text" .FlightNumber ) }}
+    {{template "table-cell-paragraph" (extendDict $props "TextColor" "#000000" "Text" .Gate ) }}
+    {{template "table-cell-paragraph" (extendDict $props "TextColor" "#000000" "Text" .Class ) }}
+
+    <table-cell border-width-bottom="1" indent="0" background-color="#ffffff">
+        <table columns="2" column-widths="0.1 0.9">
+            <table-cell indent="0" vertical-align="middle">
+                <image src="path('templates/res/seat_icon.png')" fit-mode="fill-width" margin="0"></image>
+            </table-cell>
+
+            {{template "table-cell-paragraph" (extendDict $props "Margin" "0" "BorderBottomSize" 0 "TextColor" "#ff0000" "Text" .Seat ) }}
+        </table>
+    </table-cell>
+
+    {{template "table-cell-paragraph" (extendDict $props "BorderBottomSize" 0 "Margin" "5 0 0 0" "Font" "helvetica" "FontSize" 10 "TextColor" "#000000" "Text" "Date") }}
+    {{template "table-cell-paragraph" (extendDict $props "Text" "Boarding time till") }}
+    {{template "table-cell-paragraph" (extendDict $props "Text" "Departure time") }}
+    {{template "table-cell-paragraph" (extendDict $props "Text" "Arrive") }}
+
+    {{template "table-cell-paragraph" (extendDict $props "Margin" "0" "Font" "helvetica-bold" "FontSize" 12 "Text" .Date ) }}
+    {{template "table-cell-paragraph" (extendDict $props "Text" .BoardingTime ) }}
+    {{template "table-cell-paragraph" (extendDict $props "Text" .DepartureTime ) }}
+    {{template "table-cell-paragraph" (extendDict $props "Text" .ArrivalTime ) }}
+</table>
+{{end}}
+
+{{$props := dict "Colspan" 1 "Rowspan" 1 "Align" "left" "VerticalAlign" "top" "BackgroundColor" "#ffffff" 
+    "BorderColor" "#000000" "BorderTopSize" 0 "BorderBottomSize" 0 "Indent" 0 
+    "Margin" "3 0 5 0" "LineHeight" 1 "Font" "helvetica" "FontSize" 10 "TextColor" "#000000"}}
+
+{{template "boarding-pass-form" . }}
+
+<table columns="2" column-widths="0.5 0.5">
```

Comment:
There's no need to specify the `column-widths` attribute. They will have equal sizes by default.